python/lsst/jointcal/jointcal.py

```diff
@@ -156,7 +156,9 @@ class JointcalTaskConnections(pipeBase.PipelineTaskConnections,
         multiple=True,
     )
     inputVisitSummary = pipeBase.connectionTypes.Input(
-        doc="Visit summary tables built from the calexps for these observations.",
+        doc=("Per-visit consolidated exposure metadata built from calexps. "
+             "These catalogs use detector id for the id and are sorted for "
+             "fast lookups of a detector."),
```

Comment:
Because this is Input, I would phrase it as "...and must be sorted for fast ...", since that's a requirement on the files, not something this package guarantees.